tests/stack_integrity_thread.cc

```diff
@@ -89,8 +108,7 @@ void test_stack_invalid_on_call(bool *outTestFailed,
 {
 	threadStackTestFailed = outTestFailed;
 
-	__asm__ volatile("ccleartag      csp, csp\n"
-	                 "csh            zero, 0(cnull)\n");
+	CALL_CHERI_CALLBACK(fn, "move a0, %1\nccleartag csp, csp\n", 0);
```

Comment:
Is the `move` instruction useless? Might be better with a comment.

Reply:
That's annoying. I wanted to have one macro for both callsites to avoid repeating code, but I need operand in the first and not in the later. The `move a0, %1` is there just to use the operand.

Do you have a way to make it generic without it?